PowerEditor/src/NppCommands.cpp

```diff
@@ -2855,7 +2855,7 @@ void Notepad_plus::command(int id)
 		case IDM_CMDLINEARGUMENTS:
 		{
 			// No translattable
-			::MessageBox(NULL, COMMAND_ARG_HELP, TEXT("Notepad++ Command Argument Help"), MB_OK);
+			::MessageBox(_pPublicInterface->getHSelf(), COMMAND_ARG_HELP, TEXT("Notepad++ Command Argument Help"), MB_OK | MB_APPLMODAL);

```

Comment:
Personally, I don't like modal dialog.
It's not harmful if users want n windows of help.

Reply:
@donho The problem is it is easy to get the Help window open and BEHIND the main window (or other windows).  Then shortcuts in the main window DON'T work. This happened to me the other day and it took quite some time to understand what was happening.  Read https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4067#issuecomment-519245178 for details.  This PR is a good fix; I tested it. :)

Reply:
Did you read Scott's comment?

> In addition to the problems already noted, if the Notepad++ Command Argument Help window remains open and the user returns to editing within a Notepad++ editor tab view, odd things happen.
> 
> For an example of one, if the user presses ctrl+s to save the active document, it is not saved and a DC3 character is placed at the current position in the editor buffer. Another example is ctrl+w to close the active document; in this case an ETB character is inserted into the text instead. Closing out the open Notepad++ Command Argument Help window restores normal functionalities for these two examples.

Reply:
Finished testing the artifact build.  No problems found.  @donho Thank you for reconsidering. @xylographe Thanks for doing this code change.  :)